examples/Microsoft.Spark.CSharp.Examples/Sql/Streaming/StructuredNetworkCharacterCount.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Spark.Sql;
+using static Microsoft.Spark.Sql.Functions;
+
+namespace Microsoft.Spark.Examples.Sql.Streaming
+{
+    /// <summary>
+    /// This an example of using a UDF with streaming processing.
+    ///
+    /// You can set up the data source as follow in a separated terminal:
+    /// `$ nc -lk 9999`
+    /// to start writing standard input to port 9999.
+    /// </summary>
+    internal sealed class StructuredNetworkCharacterCount : IExample
+    {
+        public void Run(string[] args)
+        {
+            // Default to running on localhost:9999
+            string hostname = "localhost";
+            var port = 9999;
+
+            // User designated their own host and port
+            if (args.Length == 2)
+            {
+                hostname = args[0];
+                port = int.Parse(args[1]);
+            }
+
+            SparkSession spark = SparkSession
+                .Builder()
+                .AppName("Streaming example with a UDF")
+                .GetOrCreate();
+
+            DataFrame lines = spark
+                .ReadStream()
+                .Format("socket")
+                .Option("host", hostname)
+                .Option("port", port)
+                .Load();
+
+            // UDF to produce an array
+            // Array includes: original string, length of original string
+            Func<Column, Column> udfArray =
+                Udf<string, string[]>((str) => new string[] { str, str + " " + (str.Length).ToString() });
+            DataFrame arrayDF = lines.Select(Explode(udfArray(lines["value"])));
```

Comment:
I thought it looked nicer to have each element of the array on its own row, so that's why I included `Explode` (it would still function fine without it).

Reply:
Just curious what the output is with and without this `Explode`

Reply:
With `Explode`:
+----+
| col|
+----+
|  hi |
|hi 2|
+----+

Without `Explode`:
+-------------------------------------------------+
|System.String[] <Run>b__0_0(System.String)(value)|
+-------------------------------------------------+
|                                                          [hi, hi 2]|  |
+-------------------------------------------------+

Reply:
Marking this as resolved, let me know if you have any other questions/suggestions!